app/Http/Resources/UserResource.php

```diff
@@ -19,6 +22,8 @@ public function toArray($request)
             'username' => $this->username,
             'full_display_name' => $this->full_display_name,
             'rewards' => $this->rewards,
+            'character' => new CharacterResource(Character::where('user_id', $this->id)->get()->first()),
```

Comment:
Kan je de character niet meteen meegeven met de user?
bijv:
in de controller: $user->with('Characters')
in de resource: 'character' => new CharacterResource($this->character)

 zo houd je namelijk het ophalen van data op 1 plek